### PR TITLE
remove autoInfusing iifes

### PR DESCRIPTION
**Pull request by vendethiel**

### CODE REVIEW

**Comment by satyr** [comment]
Context: gkz/livescript#423

File: src/ast.co [hunk 1]
```diff
@@ -110,7 +110,7 @@
   carp: ->
     throw SyntaxError "#it on line #{ @line or @traverseChildren -> it.line }"
 
-  # Defines delegaters.
+  # Defines delegators.
```
Comment (satyr) [inline src/ast.co:113]: Why?

Comment (vendethiel) [inline src/ast.co:113]: I tried to find it but couldn't : http://en.wiktionary.org/wiki/delegaters
but found "delegators" : 

http://en.wiktionary.org/wiki/delegators
so it seemed like a typo to me

**Comment by vendethiel** [comment]
Also, if you want me to "upport" the generator PR, tell me which syntax you'd like

File: src/ast.co (hunk 1 above)
Comment (satyr) [inline src/ast.co:113]: Ah thanks.

Try to avoid irrelevant changes in PR though. Every one of those lines prevents swift merging.

Comment (vendethiel) [inline src/ast.co:113]: Yeah I'll try not to do that in the future.

**Pull request merged**